AnnoyancesFilter/Other/sections/self-promo.txt

```diff
@@ -1944,6 +1944,7 @@ theguardian.com##.contributions__adblock
 !
 ! SECTION: Self-promo - Regular rules
 !
+eikaiwa.dmm.com###content div[class^="banner-"]
```

Comment:
`eikaiwa.dmm.com` consists of multple sites. I prefer two rules of
```
eikaiwa.dmm.com##.banner-default
eikaiwa.dmm.com##.banner-single-top
```

for safety.